js/util.bitcore.js

```diff
@@ -358,90 +315,74 @@ CWBitcore.signHash = function(signedTx, scriptPubKey, vin_index, cwPrivateKey, c
     }
   }
 }
-
+CWBitcore.signRawTransaction = /* BC? */
 CWBitcore.signRawTransaction2 = function(unsignedHex, cwPrivateKey, callback) {
```

Comment:
should clean up usage everywhere of `signRawTransaction2` so I can remove it....